### PR TITLE
Add events to the marker and the popup

### PR DESCRIPTION
**Pull request by alex3165**

### CODE REVIEW

**Comment by alex3165** [comment]
@philpl Can you review this 👍

File: example/server.js [hunk 1]
```diff
@@ -3,13 +3,15 @@ var webpack = require('webpack')
 var config = require('./webpack.config')
 
 var compiler = webpack(config)
+var port = 8080;
```
Comment (philpl) [inline example/server.js]: Can you replace these two lines with `process.env.X || DEFAULT`? Just for convenience

File: example/server.js [hunk 2]
```diff
@@ -3,13 +3,15 @@ var webpack = require('webpack')
...
 
 var server = new WebpackDevServer(compiler, {
   publicPath: config.output.publicPath,
   historyApiFallback: true,
   noInfo: true
 })
 
-server.listen(8080, "localhost", function() {
-  console.log('☕️  Server is listening on localhost:8080.');
+server.listen(port, host, function() {
```
Comment (philpl) [inline example/server.js]: By default `host` shouldn't probably be passed at all

File: src/marker.js [hunk 3]
```diff
@@ -12,12 +12,11 @@ export default class Popup extends React.Component {
     anchor: OverlayPropTypes.anchor,
     offset: OverlayPropTypes.offset,
     children: PropTypes.node,
```
Comment (philpl) [inline src/marker.js]: It'd be great to get all the missing callback `propTypes` in? Is there any reason why they were missing in the first place?

**Pull request merged**